VerticalSlice2D/Assets/Scripts/CombatSlider.cs

```diff
@@ -23,13 +26,20 @@ public void CheckHit()
         {
             if (hitNow) 
             {
+                //Critical Hit! spawns a blinking slider 
                 Debug.Log("<color=red>CRITICAL HIT</color>");
-                combatLogic.SliderDied(gameObject);
+                GameObject blinker = Instantiate(blinkingSlider, transform.position, Quaternion.identity);
+                blinker.GetComponent<Image>().color = Color.yellow;
+                blinker.transform.SetParent(transform.parent);
+                KillSlider();
```

Comment:
Dit stukje code wordt ook op regel 40 gedaan. Misschien zou hier een functie voor gemaakt worden en dan als parameter meegeven of het een critical hit was of niet.